talentlink-connector/processes/rest/Document.p.json

```diff
@@ -0,0 +1,56 @@
+{
+  "format" : "10.0.0",
+  "id" : "185DDFAB1E7B0FA1",
+  "kind" : "CALLABLE_SUB",
+  "config" : {
+    "data" : "talent.link.Data"
+  },
+  "elements" : [ {
+      "id" : "f0",
+      "type" : "CallSubStart",
+      "name" : "call(Long)",
+      "config" : {
+        "callSignature" : "call",
+        "input" : {
+          "params" : [
+            { "name" : "documentId", "type" : "Long" }
+          ],
+          "map" : {
+            "out.documentId" : "param.documentId"
+          }
+        }
+      },
+      "visual" : {
+        "at" : { "x" : 96, "y" : 64 }
+      },
+      "connect" : { "id" : "f4", "to" : "f3" }
+    }, {
+      "id" : "f1",
+      "type" : "CallSubEnd",
+      "visual" : {
+        "at" : { "x" : 352, "y" : 64 }
+      }
+    }, {
+      "id" : "f3",
+      "type" : "RestClientCall",
+      "name" : "Document",
+      "config" : {
+        "path" : "/candidate/document/{documentId}",
+        "clientId" : "d8e4652e-41d5-4023-90c5-5c822b8600e2",
+        "clientErrorCode" : "ivy:error:rest:client",
+        "queryParams" : {
+          "query" : "",
+          "api_key" : "ivy.var.get(\"talentlink-connector.apikey\")"
```

Comment:
re-stating the api_key query parameter on each and every rest-client call is bad practice:
- please provide a jersey feature, that adds this parameter for all calls, without actually doing it manually on the rest client activities. Just like you did it for CXF and SOAP
- if you need example code or advice how to do it, please ask for it and we can have a look at other connectors that do the same.

- [ ] JAX-RS feature on the rest-client, that adds the api_key to all calls.